includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -909,7 +909,20 @@ private function request( $params, $api, $method, $is_site_specific = true ) {
 	 * @throws API_Exception - If the account ID hasn't been set.
 	 */
 	private function request_with_level3_data( $params, $api, $method, $is_site_specific = true ) {
-		// If level3 data is not present for some reason, simply proceed normally.
+		/**
+		 * Allows the usage of level3 data to be disabled.
+		 *
+		 * @param bool $enabled Whether level3 data is enabled, true by default.
+		 * @param array  $params           - Request parameters to send as either JSON or GET string. Defaults to test_mode=1 if either in dev or test mode, 0 otherwise.
+		 * @param string $api              - The API endpoint to call.
+		 * @param string $method           - The HTTP method to make the request with.
+		 * @return bool
+		 */
+		$use_level3_data = apply_filters( 'wcpay_api_use_level3_data', true, $params, $api, $method );
```

Comment:
Not sure if there is a security or any other risk if merchant disables it by own plugin, do you? Will calling this filter in dev mode only make it safer?